ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractStateEventsReader.scala

```diff
@@ -4,59 +4,16 @@
 package com.daml.platform.store.appendonlydao.events
 
 import java.io.InputStream
-import java.sql.Connection
-import java.time.Instant
 
-import anorm.SqlParser.{binaryStream, int, long}
-import anorm._
-import com.daml.ledger.participant.state.v1.Offset
-import com.daml.lf.data.Ref
-import com.daml.platform.store.Conversions.{contractId, offset, _}
 import com.daml.platform.store.appendonlydao.events
 import com.daml.platform.store.dao.events.ContractStateEvent
 import com.daml.platform.store.serialization.{Compression, ValueSerializer}
-import com.daml.platform.store.SimpleSqlAsVectorOf.SimpleSqlAsVectorOf
 import com.daml.platform.store.LfValueTranslationCache
+import com.daml.platform.store.backend.StorageBackend.RawContractStateEvent
 
 import scala.util.control.NoStackTrace
 
 object ContractStateEventsReader {
```

Comment:
Without the actual fetching logic, the name is a bit outdated. Wouldn't `ContractStateEventsConversion` work better?

Reply:
this goes for all readers actually. I wouldn't do renames in this PR so we can keep the impact on bay, but happy to add todo to capture renaming efforts...but you know lot of improvements are on the table after this is merged

Reply:
Good point 👍